src/compas/artists/meshartist.py

```diff
@@ -405,14 +363,14 @@ def clear_faces(self):
         """
         raise NotImplementedError
 
-    def clear(self):
-        """Clear all components of the mesh.
+    # def clear(self):
```

Comment:
already defined on the base artist. was only reformulated for the mesh because of this awkward definition of `draw`, which i would now replace by what used to be `draw_mesh`, since that is the expected behaviour in almost all cases...